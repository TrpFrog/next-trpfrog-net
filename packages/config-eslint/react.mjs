// @ts-check
import { createESLintConfig } from './base.mjs'
import pluginReact from 'eslint-plugin-react'

/**
 * ESLint config for React projects.
 * @param {import('typescript-eslint').ConfigWithExtends[]} userConfig
 * @returns
 */
export const createESLintConfigReact = (...userConfig) =>
  createESLintConfig({
    files: ['**/*.tsx'],
    plugins: {
      react: pluginReact,
    },
  })
