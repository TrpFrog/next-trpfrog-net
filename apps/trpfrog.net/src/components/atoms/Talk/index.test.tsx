import { render } from '@testing-library/react'

import { Talk } from '.'

describe('Talk', () => {
  test('snapshot test', () => {
    const view = render(<Talk />)
    expect(view.asFragment()).toMatchSnapshot()
  })
})
