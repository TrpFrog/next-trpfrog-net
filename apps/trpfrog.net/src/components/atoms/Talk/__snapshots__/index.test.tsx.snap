// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Talk > snapshot test 1`] = `
<DocumentFragment>
  <div
    class="grid "
  />
</DocumentFragment>
`;
